test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,49 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('First Api Tests', () => {
+    it('Consume GET Service', async () => {
+        const response = await axios.get('https://httpbin.org/ip');
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data).to.have.property('origin');
+    });
+    it('Consume GET Service with query parameters', async () => {
+        const query = {
+            name: 'John',
+            age: '31',
+            city: 'New York'
+        };
+
+        const response = await axios.get('https://httpbin.org/get', { query });
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.config.query).to.eql(query);
+    });
+    it('Consume HEAD Service', async () => {
+        const response = await axios.head('https://httpbin.org/headers');
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.headers).to.have.property("content-type");
+        expect(response.data).to.eql("")
+    });
+    it('Consume PATCH Service', async () => {
+        const response = await axios.patch("https://httpbin.org/patch",{ body });
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data.data).to.eql(JSON.stringify(body));
+        expect(response.data.json).to.eql(body);
+    });
+    it('Consume PUT Service', async () => {
+        const response = await axios.put("https://httpbin.org/put",{ query });
```

Comment:
you need to define the body to be used in this method. The body should be declared in a similar way that the query is defined for the GET test.